src/rgw/rgw_common.h

```diff
@@ -754,14 +754,16 @@ struct RGWUserInfo
   uint32_t type;
   std::set<std::string> mfa_ids;
   std::string assumed_role_arn;
+  uint32_t inline_policy_size;
```

Comment:
Not required to be part of RGWUserInfo. This value is not going to be varying per user. Its constant for that setup. Refer to "rgw_list_buckets_max_chunk" key in ceph.conf file. "inline_policy_size" will be on similar lines. 
In summary do not store this info motr KVStore, and refer it runtime via conf like "cct->_conf->rgw_list_buckets_max_chunk;"